pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -83,6 +84,16 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 				},
 			},
 		}
+
+		subscriptionDoc, err := f.getSubscriptionDocument(ctx, doc.Key)
+		if err != nil {
+			return nil, err
+		}
+
+		if !f.env.IsLocalDevelopmentMode() &&
+			feature.IsRegisteredForFeature(subscriptionDoc.Subscription.Properties, "Microsoft.RedHatOpenShift/RedHatEngineering") {
+			doc.OpenShiftCluster.Properties.FeatureProfile.GatewayEnabled = true
```

Comment:
should probably make sure that this code path runs at least in INT (& PROD?) E2Es